index.html

```diff
@@ -9,16 +9,27 @@
 </head>
 
 <body>
+   <dialog class="modal-1">   
+      <div><span class="close-1">&times;</span></div>
+      <nav>
+        <ul class = "lnks">
+          <li><a href="#portfolio">Portfolio</a></li>
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+      </nav>
+      
+  </dialog>
    <header class="toolbar">
       <div class="logo">
          <h3>My Logo</h3>
       </div>
 
       <nav>
-         <ul class="nav">
-            <li>Portfolio</li>
-            <li>About</li>
-            <li>Contact</li>
+         <ul class="desktop-nav">
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Kindly revisit the learning materials for this section to help you fix the pending issues. 
- I can see that the popup does not cover the whole screen just as it is on the template you chose, and as suggested by the last reviewer.  This [Link](https://dev.to/javascriptacademy/responsive-navigation-bar-with-mobile-menu-using-html-css-2hp) might come in handy to get a heads up 

- Kindly adjust the left margin space in your design in relation to the template prrovided. [This detailed Link](https://www.freecodecamp.org/news/how-to-build-a-responsive-navigation-bar-with-dropdown-menu-using-javascript/) has resourse that might come in handy 👍 

### Your design:
![image](https://github.com/cloud007bond/Portfolio/assets/50990925/1203ec89-47c8-46ba-8f91-d60e3dd0e589)

### Figma:
![image](https://github.com/cloud007bond/Portfolio/assets/50990925/102a9dfc-75fd-490b-863f-a8ec752b8a38)